src/QirRuntime/test/QIR-static/qir-test-math.qs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Testing.QIR.Math
+{
+    open Microsoft.Quantum.Intrinsic;
+
+    function TestSqrt() : Int
+    {
+        mutable testResult = 0;
+
+        if(           2.0 != Sqrt(  4.0) )    { set testResult = 1; }     
```

Comment:
You shouldn't need nested `ifs` here because you are not allocating qubits, early returns should compile just fine:

```
if (2.0 != Sqrt(  4.0) )    { return 1; }
if (3.0 != Sqrt(  9.0) )    { return 2; }
// etc.
```

Reply:
That would also have the beneficial effect of eliminating the `mutable` variable.

Reply:
Fixed.